test/fle2/insert/cmd.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please move the test data files into the `test/data` directory?

Reply:
Done